plugin/pki/util.go

```diff
@@ -247,18 +247,24 @@ func isTokenRefreshNeeded(b *backend, ctx context.Context, storage logical.Stora
 	if err != nil {
 		return false, "", err
 	}
-	return secretEntry.NextRefresh.Before(time.Now()), secretEntry.RefreshToken2, nil
+	now := time.Now()
```

Comment:
Is it required to provide the time?, is it not displayed by the Logger itself?

Reply:
@marcos-albornoz I printed this way while debugging so it was easier, but now this has already been removed Marcos